javascript.js

```diff
@@ -0,0 +1,20 @@
+// Set variavles
```

Comment:
Typo inside comment in token 'Set'. Possible corrections:get (81),set (70),let (18)